boards/shields/apollo4p_evb_disp_shield/apollo4p_evb_disp_shield.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (c) 2023 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		accel0 = &adxl362;
+	};
+};
+
+&iom1 {
```

Comment:
https://docs.zephyrproject.org/latest/hardware/porting/shields.html#board-compatibility

https://github.com/zephyrproject-rtos/zephyr/blob/ef5d0fd291568e39d55ef95792e217146d6c2043/boards/arm/apollo4p_evb/apollo4p_evb.dts#L37-L51

No "well-known connectors" -> not acceptable PR.